__tests__/tests.ts

```diff
@@ -336,6 +336,59 @@ describe('htmlbars-inline-precompile', function () {
     `);
   });
 
+  it('works with renamed scope', function () {
+    plugins = [
+      [
+        HTMLBarsInlinePrecompile,
+        {
+          compiler,
+
+          targetFormat: 'wire',
+          enableLegacyModules: [
+            'ember-cli-htmlbars',
+            'ember-cli-htmlbars-inline-precompile',
+            'htmlbars-inline-precompile',
+          ],
+        },
+      ],
+    ];
+
+    /**
+     * This scenario happens within Babel (somewhere) when
+     * import { Setup as Foo } from '...' is done.
+     * The import alias is undone, and the aliasing is moved to the usage location.
+     *
+     * Being wrapped in setComponentTemplate is required, else the scope bag is correct
+     * (ie: [Setup]).
+     */
+    let code = `
+      import { Setup } from './foo.js';
+      import { precompileTemplate } from '@ember/template-compilation';
+      import { setComponentTemplate } from '@ember/component';
+      import templateOnly from '@ember/component/template-only';
+
+      export default setComponentTemplate(precompileTemplate("<Foo />", {
+        strictMode: true,
+        scope: () => ({
+          Foo: Setup
+        })
+      }), templateOnly());
+    `;
+
+    let transformed = transform(code);
+
+    /**
+     * Have to choose our line because `moduleName` differs per-machine.
```

Comment:
Please take a look at all the other wire format tests and use the helper (`normalizeWireFormat`) and assertions that they use.

Reply:
done :tada: